DQM/DTMonitorModule/src/DTLocalTriggerLutTask.cc

```diff
@@ -265,12 +300,44 @@ void DTLocalTriggerLutTask::searchDccBest( std::vector<L1MuDTChambPhDigi> const*
     int st   = trigIt->stNum();
     int qual = trigIt->code();
 
-    if(qual>trigQualBest[wh+3][st][sec] && qual<7) {
-      trigQualBest[wh+3][st][sec]=qual;
-      trigBest[wh+3][st][sec] = &(*trigIt);
+    if(qual>trigQualBestIn[wh+3][st][sec] && qual<7) {
```

Comment:
I gather "3" is a sign that "wh" goes from -3 to 2? Especially since it occurs so many times, please make a const variable with a meaningful name instead of hardwiring 3.